dom.go

```diff
@@ -2072,6 +2194,11 @@ func (ctx *CanvasRenderingContext2D) Restore() {
 	ctx.Call("restore")
 }
 
+//Hit regions
+//addHitRegion
+//removeHitRegion
+//clearHitRegions
```

Comment:
It's completely okay to leave them as a TODO for the future.

You should only implement them if you want to or need them yourself. I don't need them right now.

Either way, it's better to leave out for another PR, to avoid making this one larger.